package.json

```diff
@@ -130,12 +130,12 @@
     "json-stringify-pretty-compact": "~4.0.0",
     "tslib": "~2.8.1",
     "vega-event-selector": "~3.0.1",
-    "vega-expression": "~5.1.1",
-    "vega-util": "~1.17.2",
+    "vega-expression": "~5.1.2",
+    "vega-util": "~1.17.3",
     "yargs": "~17.7.2"
   },
   "peerDependencies": {
-    "vega": "^5.24.0"
+    "vega": "^5.31.0"
```

Comment:
Do we require this? I'd say let's be conservatives and only update the constraint here if we know of a functionality we need.